### PR TITLE
fix firefox & crossbrowser

### PR DESCRIPTION
**Pull request by SanthoshJeeva440**
1) fix- fireFox launcher
2) fix- crossBrow.xml 
3) optimize page object with singleton design pattern

### CODE REVIEW

File: suiteTest.xml [hunk 1]
```diff
@@ -3,8 +3,8 @@
 <suite name="All Test Suite" >
     <test verbose="2" name="search hotel" preserve-order="true" thread-count="1" parallel="classes">
         <classes>
-            <class name="e2e.SearchHotel"/>
             <class name="e2e.login"/>
```
Comment (syedth) [inline suiteTest.xml:6]: @SanthoshJeeva440 "e2e" should be added as an overall package rather adding each single classes

File: src/test/java/modules/SearchHotel.java [hunk 2]
```diff
@@ -1,25 +1,31 @@
 package modules;
 import baseClass.SeleniumMethods;
 import pageObject.HotelSearchPojo;
+import pageObject.PageObjectManager;
+
 public class SearchHotel{
 
-    public static HotelSearchPojo hotel = new HotelSearchPojo();
+//    public static HotelSearchPojo hotel = new HotelSearchPojo();
```
Comment (syedth) [inline src/test/java/modules/SearchHotel.java]: @SanthoshJeeva440 Please remove the commented code

**Review by syedth — COMMENTED** [review]
please check the comments

File: suiteTest.xml (hunk 1 above)
Comment (SanthoshJeeva440) [reply suiteTest.xml:6]: @syedth added the e2e package in the crossBrow.xml file

File: src/test/java/modules/SearchHotel.java (hunk 2 above)
Comment (SanthoshJeeva440) [reply src/test/java/modules/SearchHotel.java]: @syedth the commented code are removed

**Pull request merged**